views/components/application-card/macro.njk

```diff
@@ -0,0 +1,114 @@
+{% from "components/icons.njk" import iconBin %}
+{% from "components/progress-marker/macro.njk" import progressMarker with context %}
+
+{% macro statusMessage(status) %}
+    {% if status === 'ineligible' %}
+        Ineligible
+    {% elseif status === 'complete' %}
+        Submitted for consideration
+    {% else %}
+        Not submitted
+    {% endif %}
+{% endmacro %}
+
+{% macro applicationOverview(application) %}
+    <dl class="o-definition-list o-definition-list--compact u-margin-top">
+        <dt>Proposed start date</dt>
+        <dd>
+            {% set dateParts = application.application_data['project-start-date'] %}
```

Comment:
This made me realise that we need a way to use the `displayFormat` logic for the form model for these application summaries too, as this is duplicating that logic in templates which is much harder to test.

Reply:
Yeah... wonder how we can do that.

Reply:
My thinking is that each "application" in this list should be a copy of the full form model rather than the raw application data, so the same process we go through to "enhance" the data in steps but per application.